doc/security/security-best-practice-en.md

```diff
@@ -0,0 +1,83 @@
+
+For the federated learning without third-party arbiter, its security assumption is: each participant in FATE is an honest, curious and independent security domain, and the data interaction between participants can ensure that the privacy data of each security domain is not leaked. For the federated learning with third-party arbiter, the security assumption is: the federal learning participants are honest, curious and independent security domains, and the arbiter is honest and curious for all participants. That is, the arbiter will comply with the FATE protocol's execution, but may retain or inference aggregation stage intermediate data. When multiple arbiters perform aggregation and arbitration operations, they must not collude with each other.
+
+Security models of FATE need to use security protocols such as secure multi-party computing to protect data security during data interaction stage between different participants. FATE security protocols include Paillier Encryption, RSA Encryption, Hash Factory, DH Key Exchange, SPDZ, OT, etc.
+
+## 2. Network Security Suggestions
+
+For the participating nodes in FATE, client_authentication and permission components need to be used to realize identity authentication function between nodes.
+
+In the network transmission process involved in FATE, such as HTTPS, Flask, gRPC, etc., SSL/TLS protocol is used to protect the security of data transmission. In particular, SSL/TLS is required for network transmission in a public network environment. TLS key strength complies with RFC8446 section9.1.
+
+FATE usually expose one Network port: 9370，network data package of other ports need to be treated with caution.
+
+## 3. Security Protocol Configuration Recommendations
+
+The federated learning security protocol that FATE supports are: Paillier Homomorphic Encryption, Hash Factory, DH Key Exchange, SecretShare MPC Protocol, Oblivious Transfer. The table below shows the security configuration recommendation. 
+
+| Protocol Name        | Algorithm Classification   | Security Configuration Recommendation |
+| :-------------:| :----------: | :------------: |
+| Paillier Encryption|asymmetric encryption algorithm   | Refer to RSA algorithm requirement, minimum key length (module length) is 1024 bits, 2048 bits is recommended |
+| RSA Encryption     |asymmetric encryption algorithm   |minimum key length (module length) is 1024 bits, 2048 bits is recommended  |
+| ECDH Encryption    | asymmetric encryption algorithm based on elliptic curve |        key length 256bits   |
+| Diffie Hellman Key Exchange|key exchange algorithm| minimum key length 1024bits, 2048bits recommended      |
+| Hash Factory       |hash algorithm        |sha256，sm3 are recommended            |
+| SecretShare MPC Protocol (SPDZ) | secret share algorithm  | participants in secret share cannot collusion with each other, and two participants at least; use trusted third party or Paillier Encryption to generate multiply triples |
```

Comment:
Grammar issue: "cannot collusion with each other" -> cannot collude with each other
(maybe a global grammar check is needed)